src/components/history-page/pin-button/pin-button.tsx

```diff
@@ -17,6 +17,14 @@ export interface PinButtonProps {
   className?: string
 }
 
+/**
+ * Renders a button to pin a note.
+ *
+ * @param isPinned If the note is already pinned
+ * @param onPinClick The callback, that is fired when the button is clicked
+ * @param isDark If the button should be rendered in dark or not.
+ * @param className Additional classes directly given to the button
```

Comment:
While we USE this button to pin a note, there is no code inside that is "history pinning" exclusive.